src/app/subject/2-d-art-and-design/page.tsx

```diff
@@ -2,10 +2,10 @@ import { Accordion } from "@/components/ui/accordion";
 import Footer from "@/components/ui/footer";
 import Navbar from "@/components/ui/navbar";
 import { type Subject } from "@/types";
-import SubjectBreadcrumb from "@/app/_components/subject-breadcrumb";
-import SubjectSidebar from "../../_components/subject-sidebar";
-import TableOfContents from "../../_components/table-of-contents";
-import UnitAccordion from "@/app/_components/unit-accordion";
+import SubjectBreadcrumb from "@/components/ui/subject-homepage/subject-breadcrumb";
+import SubjectSidebar from "../../../components/ui/subject-homepage/subject-sidebar";
+import TableOfContents from "../../../components/ui/subject-homepage/table-of-contents";
```

Comment:
use @/
Also, components shouldn't all be under the components/ui section. These components should probably be under the components/subject/... folder

Reply:
Yea, I reorganized the files.